### ISSUE TITLE
Hint how to unlock exercises

### ISSUE THREAD

**Issue by szabgab** [exercism/exercism#6051]
TLDR: When hovering over a locked exercise the UI should give a hint on how to unlock it.

As a person new to Exercism I did a few exercises, but there are still a lot more that are unlocked.
For example from the syllabus I went to the [basic set of exercises in python](https://exercism.org/tracks/python/concepts/basics)
I see that `Guido's lasagna` is completed, but 4 other practice exercises are locked.

I don't know how to unlock them and when I hover over the locked exercises I only see the lock. Pretty frustrating.

I think there should be some hint (e.g. a link to some explanation) why are these locked and what does the user need to do to unlock them.

**Comment by taiyab** [comment]
@szabgab Yep, makes sense. It's something we intended to do, but just didn't get round to it.

I'll put something together.

**Comment by taiyab** [comment]
@iHiD Two things.

1. Punchcard tooltip can have a line underneath on locked exercises stating what needs to be done:
![image](https://user-images.githubusercontent.com/5337876/147935761-468be43d-ee2b-4be7-8afd-f66855cc2949.png)

2. We do a tooltip on the Exercises tab for locked exercises with this: 
![Uploading image.png…]()

We should also consider giving people access to the locked exercise page, with links to the exercises they need to complete to unlock it.

Posted design references in Slack.

**Comment by colincullen** [comment]
Tayiab,

The Punchcard tooltips you mention do not fully display. They do not show the full card.

![Screen Shot 2022-03-14 at 2 14 44 PM](https://user-images.githubusercontent.com/2993618/158235480-1b33ece9-9a61-40ac-8e41-ae3e89f0ea10.png)

The bottom part of the card informs the prerequisites required to unlock the locked problem.
This is frustrating. Maybe there is something else I need to know? I am viewing in Safari on a Mac, not sure if that makes any difference.

Thanks,
Col
cec@embarqmail.com

**Comment by SleeplessByte** [comment]
@colincullen this may be something with font/display settings. Do you perchance use a larger display size in your browser?

This is how it's supposed to look:

![image](https://user-images.githubusercontent.com/1964376/158240325-d4cbc3ad-70fd-4092-abef-1c69035ec4a9.png)

**Comment by taiyab** [comment]
@iHiD We should the unlock hints to this tooltip. Thoughts?

**Comment by rajlath** [comment]
I am neither getting any context helps on hovering over the locked exercises  neither there is any dot menu to click
All i am getting is a no entry icon when i hover over the locked exercises.

**Comment by zimmah** [comment]
I am running into this issue too with the #12in23 challenge.

It's suggesting me 5 challenges to do, but I have to do basically the whole python track to even unlock these 5, and I have no idea what I need to do to unlock them.

In the end I just ended up switching to exercise mode, which results in doing exercises without having all the background info needed. Still better than randomly doing assignments hoping to unlock the assignment you're actually interested in.

Showing a list of prerequisites would be great.

**Comment by murshed-83** [comment]
I have completed the available tasks but I can not find any tasks to work on. What needs to be done here?
![image](https://github.com/exercism/exercism/assets/129159022/dae360b9-3334-4ca6-ad0e-345c2ca1e70d)

**Comment by iHiD** [comment]
We fixed this pretty much everywhere so I'll close this now :)

**Comment by HendrikHuebner** [comment]
![image](https://github.com/user-attachments/assets/355a01ad-2228-4686-ab29-ce75cfffcbef)
I can't seem to find the unlocking option. Where is it located? This is the ocaml track page
